### ISSUE TITLE
Typings for `width` and `height` props

### ISSUE THREAD

**Issue by sleepyArpan** [mohitk05/react-insta-stories#295]
Hey, I just ran into an issue with the typings of `height` and `width`. I want to give both these values as `inherit` but ts errors out saying that it only expects numbers in this case. Is this intended?

**Comment by mohitk05** [comment]
Hey, thanks for raising the issue. Are you interested in solving it as well?

**Comment by sleepyArpan** [comment]
yes will raise a PR, just to be clear adding the type string would be enough I assume?
